core/src/main/web/app/mainapp/mainapp.js

```diff
@@ -1267,9 +1267,11 @@
         var keydownHandler = function(e) {
           var windowHeight = $(window).height();
           if (e.which === 33) { // page up
+            $(':focus').blur();
```

Comment:
blurring isn't right.  if you must, then move the cursor to a cell that is visible (could be a text cell or heading, not just code).